ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -9,36 +9,41 @@
     ViewData["RoutingPuzzleId"] = ViewContext.RouteData.Values["puzzleId"];
 }
 
+@using System.Text @* for StringBuilder *@
+
 @{
-    var mailtoUrl = "mailto:";
+    var mailtoUrl = new StringBuilder("mailto:");
 
     // to: Add the proper support agent as recipient
-    mailtoUrl += Model.PuzzleState?.Puzzle.SupportEmailAlias ?? (Model.Event?.ContactEmail ?? "puzzhunt@microsoft.com");
-
+    mailtoUrl.Append(Model.PuzzleState?.Puzzle.SupportEmailAlias ?? (Model.Event?.ContactEmail ?? "puzzhunt@microsoft.com"));
+    // cc: Cc the team's email address because what if this email account is a random personal one?
```

Comment:
Maybe rephrase to "CC the team's email address in case the account sending the e-mail is a personal one that's not associated with the event"?